truss/parsesvcname/parsesvcname_test.go

```diff
@@ -0,0 +1,114 @@
+package parsesvcname
```

Comment:
Test added.

There actually was a case that I hadn't covered that is somewhat related to this that was improperly handled, which was the case when a service begins with an underscore. 

However, I fixed that bug and added a test case for it.